### ISSUE TITLE
How can we extract the logits from audio?

### ISSUE THREAD

**Issue by pegahs1993** [sanchit-gandhi/whisper-jax#46]
Is there anyone who uses whisper-jax to extract logits from audio?

**Comment by sanchit-gandhi** [comment]
Hey @pegahs1993 - you'll soon be able to do this using Whisper JAX through the model + processor paradigm (see https://github.com/sanchit-gandhi/whisper-jax#model-usage), but we'll need [this PR](https://github.com/huggingface/transformers/pull/22700) merged in Transformers before we can do this in Whisper JAX

**Comment by bdwivedi6** [comment]
Hello! Has there been any progress on this front?
